### PR TITLE
Bump dependencies and plugins

### PR DESCRIPTION
**Pull request by dacrome**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -127,14 +127,14 @@
...
-            <version>9.4.1208</version>
+            <version>9.4.1209</version>
             <scope>test</scope>
         </dependency>
 
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
-            <version>5.1.39</version>
+            <version>6.0.3</version>
```
Comment (tkrille) [inline pom.xml]: According to https://dev.mysql.com/downloads/connector/j/, version 6.0.3 is still a development release. The latest GA release is 5.1.39.

Comment (dacrome) [inline pom.xml]: Ups. Changed to 5.1.39.

File: src/test/resources/context.xml [hunk 2]
```diff
@@ -51,7 +51,7 @@
         <property name="packagesToScan" value="org.osiam.client"/>
         <property name="jpaVendorAdapter">
             <bean class="org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter">
-                <property name="showSql" value="false"/>
+                <property name="showSql" value="true"/>
```
Comment (tkrille) [inline src/test/resources/context.xml]: Why this?

Comment (dacrome) [inline src/test/resources/context.xml]: Because I love SQL!!1!

**Pull request merged**